styles.css

```diff
@@ -14,7 +14,7 @@ h1 {
   font-family: Arial, Helvetica, sans-serif;
   color: blue;
   font-size: 48px;
-  font-weight:bolder; 
+  font-weight: bolder;
```

Comment:
- [ ] Please include an explanation, a "why", for each requested change. For example:

- This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

- This request lacks a "why": Why don't you use an article tag here instead of a div?